WORKSPACE

```diff
@@ -128,6 +128,12 @@ git_repository(
     remote = "https://github.com/oppia/androidsvg",
 )
 
+git_repository(
+    name = "kotlitex",
+    commit = "3e8a0804041121ece4b57099ac3de386d78e0996",
+    remote = "https://github.com/anandwana001/kotlitex",
```

Comment:
once confirmed, we will move this repo to org and update in this PR only.

Reply:
The need of creating our own version is because we need to use this in Bazel and also the repo's current min SDK is 21 and our min SDK is 19, so need to update as well. 

Also, I had removed the sample apps from the kotlitex repo, as it contains various resources like drawable, mipmap, and other colors, styles, etc which cause errors for our Bazel and Gradle for duplicate of items.